src/BasketCard.scss

```
@@ -0,0 +1,152 @@
+
+.login-form h1 {
+  font-size: 29px;
+  font-weight: 614;
+  margin-top: 0;
+  color: black;
+  font-family: 'Roboto', sans-serif;
```

скобки не нужны (исправь везде)
```suggestion
  font-family: Roboto, sans-serif;
```

у тебя слишком часто повторяется эта строка кода, вынеси в общий контейнер